src/features/LoginByRoleMemberTests.feature

```diff
@@ -28,14 +28,14 @@ Feature: Logging with Member
 		And user can see "Program Administration" in navigation
 
 	@BI-845
-	Scenario: Logging in as a breeder of one program
+	Scenario: Logging in as a Read Only of one program
```

Comment:
Seems like this was just a labeling issue of the scenario like the last one

Reply:
Yes, I figured I might as well fix the labels since I was changing them anyway.